code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -45,8 +45,7 @@
 		L.rad_act(20)
 
 /datum/weather/rad_storm/end()
-	if(..())
```

Comment:
why are you changing the behaviour of the parent return ?

Reply:
Coderbus advised I do this as the change I made _shouldn't_ make any actual changes? It seems like some funky code. Am I breaking something here?